subworkflows/nf-core/fasta_index_dna/meta.yml

```diff
@@ -0,0 +1,61 @@
+name: "fasta_index_dna"
+description: |
+  Generate aligner index for a reference genome.
+  Please note, this workflow requires input CHANNELS. Input values will cause errors
+keywords:
+  - fasta
+  - index
+  - bwamem
+  - bwamem2
+  - dragmap
+  - snapaligner
+modules:
+  - bowtie2/build
+  - bwa/index
+  - bwamem2/index
+  - dragmap/hashtable
+  - snapaligner/index
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing reference information
+        e.g. [ id:'test' ]
+  - fasta:
+      type: file
+      description: Reference genome fasta file
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing altliftover information
+        e.g. [ id:'test' ]
+  - altliftover:
+      type: file
+      description: |
+        Sam formatted liftover file for the reference genome alt contigs
+        See: https://github.com/lh3/bwa/blob/master/README-alt.md
+        Download from: https://sourceforge.net/projects/bio-bwa/files/bwakit
```

Comment:
@SPPearce, see here

Reply:
Only used for snap then?

Reply:
Only used by `snap` during index generation. `bwamem2` reads the file as is and `bwa` uses it in the `bwakit` postalt script AFAIK